README.md

```diff
@@ -125,11 +139,11 @@ bin/rails generate component Example title content --template-engine slim
 
 #### Implementation
 
-An `ActionView::Component` is a Ruby file and corresponding template file (in any format supported by Rails) with the same base name:
+A `ViewComponent` is a Ruby file and corresponding template file (in any format supported by Rails) with the same base name:
 
 `app/components/test_component.rb`:
 ```ruby
-class TestComponent < ActionView::Component::Base
+class TestComponent < ViewComponent::Base
   validates :content, :title, presence: true
```

Comment:
Is this possible to do with this new change? (taking into account that this example is using `ViewComponent::Base` instead of `ActionView::Component::Base`)